students/phrost/Lesson03/Exercises/String.py

```diff
@@ -0,0 +1,72 @@
+### Lesson_3 - String Format
+
+### Task 1 ###
+""" Origignal - ( 2, 123.4567, 10000, 12345.67) """
+""" Desired - 'file_002 :   123.46, 1.00e+04, 1.23e+04'"""
+print("\nTask 1:\n")
+print("file_{:03d} : {:10.2f}, {:.2e}, {:.3g}".format(2, 123.4567, 10000, 12345.67))
+
+###Task 2###
+""" Alternate Method """
+print("\nTask 2:\n")
+alternate = (2, 123.4567, 10000, 12345.67)
+
+def sorted(lst):
+    return "file_{0:03d}:      {1:.2f}, {2:.2e}, {3:.2e}".format(lst[0],lst[1],lst[2],lst[3])
+print(sorted(alternate))
+
+###Task 3 ###
+"""In [20]: formatter((2,3,5))"""
+"""Out[20]: 'the 3 numbers are: 2, 3, 5'"""
+
+"""In [21]: formatter((2,3,5,7,9))"""
+"""Out[21]: 'the 5 numbers are: 2, 3, 5, 7, 9'"""
+
+print("\nTask 3:\n")
+def formatter(in_tuple):
+    fstring = ""
+    for _ in range(0, len(in_tuple)):
+        fstring += '{:d}, '
```

Comment:
since you're not using the index from range() you can just loop over the tuple directly
```
for _ in in_tuple:
    ...
```